ansible/roles/packages/tasks/install-centos.yaml

```diff
@@ -0,0 +1,192 @@
+---
```

Comment:
This new file is almost identical to `install-redhat.yaml`, can we keep it combined?

Maybe we can set a new fact, `use_versionlck` based on the OS and then use that to determine if it should be installed and run in these plays instead of `versionlocklist.rc`.

![image](https://user-images.githubusercontent.com/4969231/151008738-d1517355-6098-4ba8-9e01-c00a11402803.png)

Reply:
+1. 
 However versionlock and yum-utils plugins are not allowed to upgrade only for RHEL 8.x in offline mode since they upgrade dependent packages from the *offline* repo that are protected. Protected packages are allowed to upgrade if they come from the same RHEL redhat base repo that installed them.

So I am thinking to use following conditions to set the fact
```yaml
  when: 
    - not (offline_mode_enabled and ansible_distribution == "RedHat" and ansible_distribution_major_version == '8')
```
wdyt?

Reply:
remved centos specific task file. added following fact
```yaml
# yum-plugin-versionlock is deprecated in rhel 8 and replaced with python3-dnf-plugin-versionlock.
# python3-dnf-plugin-versionlock has hard dependency on dnf and yum versions which are protected packages on RedHat 8.x and not allowed to upgraded by the OS
- name: set use package versionlock fact
  set_fact:
    versionlock_plugin_enabled: "{{ not (offline_mode_enabled and ansible_distribution == 'RedHat' and ansible_distribution_major_version == '8') }}"

```